src/test/java/teammates/test/cases/browsertests/InstructorSubmissionAdjustmentUiTest.java

```diff
@@ -38,7 +38,7 @@ protected void prepareTestData() {
 
         removeAndRestoreDataBundle(testData);
     }
-
+    
```

Comment:
Once these 4 trailing spaces are deleted, this entire file should disappear from the changeset.